conf/mini_provisioner/v2/ha.config.tmpl.1-node

```diff
@@ -24,3 +24,7 @@ server_node:
     storage_set_id: TMPL_STORAGE_SET_ID
     s3_instances: '1'
     type: TMPL_TYPE
+    bmc:
```

Comment:
It is only to create stonith resource 

so do we need it config

Reply:
To create stonith resource we need bmc info right.

Reply:
@shindeam We need to update other template files as well; init, test, reset, cleanup, etc.

Reply:
Done